### DISCUSSION TITLE
How to using Tailwind on Single-Spa

### DISCUSSION THREAD

**Discussion by fauziUI** [tailwindlabs/tailwindcss#8155] [Help] [answered]
please help me, I need some idea about URL

"@project/styleguide": "https://xxxxxxxxx/project-styleguide.js" on root-config file

or step-step to running Tailwind on single spa project

**Comment by ThaddeusJiang** [comment, marked as answer]
Hi @fauziUI 

https://github.com/react-microfrontends/styleguide

**Reply by fauziUI** [reply]
done, case closed thank you

**Reply by SergSpice** [reply]
That styleguide repository is from 2 years ago, and many single-spa and tailwind versions behind, this is not a modern answer and it doesn't work

**Reply by smfoisal** [reply]
The provided example is an old one. It's not working anymore. Is there any plan to add tailwind v3 support?
